packages/python-packages/api-stub-generator/apistub/__init__.py

```diff
@@ -33,4 +33,5 @@ def console_entry_point():
         )
     with open(out_file_path, "w") as json_file:
         json_file.write(json_tokens)
+    print("Created json file {0}".format(out_file_path))
```

Comment:
Should this be a raw print statement or a logging statement?

Reply:
I will change it to logging statement

Reply:
Removed this line.